build.gradle.kts

```diff
@@ -72,8 +72,8 @@ tasks.register("checkAll") {
 }
 
 registerSubModuleAggregationTask("assembleLibraries", "assemble")
-registerSubModuleAggregationTask("unitTestRelease", "testReleaseUnitTest", exceptions = setOf("dd-sdk-android-trace"))
-registerSubModuleAggregationTask("unitTestDebug", "testDebugUnitTest", exceptions = setOf("dd-sdk-android-trace"))
```

Comment:
there is no test task on the CI for Otel module?

Reply:
there is actually if you check that `registerSubModuleAggregationTask` it is doing this automatically for all the modules but let me double check in gitlab

Reply:
then another question :) what changed so that it allows us to run tests without these exceptions?

Reply:
:), yes good question there, I think I added this exception when I ported the Java code as an intermediary change and I forgot to remove it :(. I checked the data I changed those lines and it make sense.